.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Primary owner should be listed first in list of global owners, followed by any secondary owners
+
```

Comment:
Can you add codeowners to this? Suggest to use same codeowners as our internal vpc module

Reply:
added